keyboards/tweetydabird/lotus58/keymaps/vial/keymap.c

```diff
@@ -0,0 +1,113 @@
+	____, ____, ____, ____, ____, ____, KC_PGDN,        KC_PGUP, ____, ____, ____, ____, ____, ____,
+					  ____, ____, ____, ____,      	 ____, ____, ____, ____
+    ),
+
+    [2] = LAYOUT(
+    ____, ____, ____, ____, ____, ____, ____,        ____, ____, ____, ____, ____, ____, ____,
+	____, ____, ____, ____, ____, ____,                    ____, ____, ____, ____, ____, ____,
+	____, ____, ____, ____, ____, ____,                    ____, ____, ____, ____, ____, ____,
+	____, ____, ____, ____, ____, ____, ____,        ____, ____, ____, ____, ____, ____, ____,
+					  ____, ____, ____, ____,      	 ____, ____, ____, ____
+    ),
+
+    [3] = LAYOUT(
+    QK_BOOT, ____, ____, ____, NK_TOGG, AC_TOGG, ____,        ____, ____, ____, ____, RGB_VAI, RGB_VAD, RGB_TOG,
+	QK_RBT, ____, ____, ____, ____, ____,                    ____, ____, ____, RGB_HUI, RGB_HUD, RGB_M_P,
+	____, ____, ____, ____, ____, ____,                    ____, ____, ____, RGB_SAI, RGB_SAD, RGB_M_B,
+	____, ____, ____, ____, ____, ____, ____,        ____, ____, ____, ____, RGB_SPI, RGB_SPD, RGB_M_R,
+					  ____, ____, ____, ____,      	 ____, ____, ____, ____
+    ),
+};
+
+#if defined(ENCODER_MAP_ENABLE)
+const uint16_t PROGMEM encoder_map[][NUM_ENCODERS][2] = {
+    [0] = { ENCODER_CCW_CW(KC_VOLD, KC_VOLU), ENCODER_CCW_CW(KC_VOLD, KC_VOLU) },
+    [1] = { ENCODER_CCW_CW(KC_TRNS, KC_TRNS), ENCODER_CCW_CW(KC_TRNS, KC_TRNS) },
+    [2] = { ENCODER_CCW_CW(KC_TRNS, KC_TRNS), ENCODER_CCW_CW(KC_TRNS, KC_TRNS) },
+    [3] = { ENCODER_CCW_CW(KC_TRNS, KC_TRNS), ENCODER_CCW_CW(KC_TRNS, KC_TRNS) }
+};
+#endif
+
+#ifdef OLED_ENABLE
+
+static void print_status_narrow(void) {
+	// Create OLED content
+    oled_write_P(PSTR("\n"), false);
+	oled_write_P(PSTR(""), false);
+    oled_write_P(PSTR("Lotus -58-"), false);
+	oled_write_P(PSTR("\n"), false);
+
+    // Print current layer
+    oled_write_P(PSTR("Layer"), false);
+    switch (get_highest_layer(layer_state)) {
+        case 0:
+            oled_write_P(PSTR("-Base\n"), false);
+            break;
+        case 1:
+            oled_write_P(PSTR("-Func\n"), false);
+            break;
+        case 2:
+            oled_write_P(PSTR("-Num \n"), false);
+            break;
+        case 3:
+            oled_write_P(PSTR("-Sys \n"), false);
+            break;
+        default:
+            oled_write_P(PSTR("Undef"), false);
+    }
+	
+	// Display capslock
+    oled_write_P(PSTR("\n"), false);
+    led_t led_usb_state = host_keyboard_led_state();
+    oled_write_ln_P(PSTR("Caps- lock"), led_usb_state.caps_lock);
+	
+#ifdef AUTO_SHIFT_ENABLE
+
+	bool autoshift = get_autoshift_state();
+	oled_write_P(PSTR("\n"), false);
+	oled_write_P(PSTR("Auto-Shift"), autoshift);
+	oled_write_P(PSTR("\n"), false);
+	
+#endif
+
+}
+
+oled_rotation_t oled_init_user(oled_rotation_t rotation) {
+	return OLED_ROTATION_270;
+}
+
+bool oled_task_user(void) {
+    if (is_keyboard_master()) {
+        print_status_narrow();
+    }
```

Comment:
Terribly sorry, I steered you slightly wrong here with my earlier suggestion. As you're printing the same thing on both sides, the entire `if` statement isn't necessary.
```suggestion
    print_status_narrow();
```